mlflow/entities/span.py

```diff
@@ -345,6 +345,15 @@ def end(self):
     def from_dict(cls, data: Dict[str, Any]) -> "Span":
         raise NotImplementedError("The `from_dict` method is not supported for the LiveSpan class.")
 
+    def to_completed_span(self) -> "Span":
```

Comment:
Can we rename this function? Because there's no docstring saying `Span` is a completed span. Or can we instead create classmethod `from_live_span` in Span?

Reply:
Makes sense, what about `to_immutable_span`? I think we should keep the method in subclass so the parent class (`Span`) should not depends on the child (`LiveSpan`) in general.

Reply:
`to_immutable_span` sounds good! Could you also add a comment at the place where this method is called? Because I assume most of people seeing the code first won't know it's a LiveSpan 😅

Reply:
Absolutely! Updated:)